testsuite/render-mx-sheen/sheen.osl

```diff
@@ -16,7 +16,7 @@ sheen
   )
 {
     if (N.x < 0)
-        Ci = sheen_bsdf (N, Cs, roughness * 2, "mode", 1);
+        Ci = sheen_bsdf (N, Cs, sqrt(roughness), "mode", 1);
```

Comment:
Are you sure this is correct? I thought the external parameterisation was roughness (alpha in the Zeltner implementation) and that only the underlying SGGX model uses sigma. The table itself (and my analytic refit) should be parameterised by alpha, not sigma.

Reply:
When looking at the wedges of both sheen models, the sqrt seemed to bring them closer together. This part is just for the testsuite where its handy to see both models at once on the same sphere.

The materialx closure just exposes the "raw" parameter in any case (just like GGX).

Reply:
Okay, makes sense.